config/routes.rb

```diff
@@ -9,6 +9,8 @@
   devise_for :users, controllers: { registrations: 'registrations' }
   resources :users, only: [:show]
   
+  # get '/:locale' => 'home#index'
+
   root to: 'home#index'
   match 'preview',          to: 'home#preview', via: :get
```

Comment:
Any particular reason why preview, about, report, report_redirect, and root aren't under the locale scope?